App.tsx

```diff
@@ -110,7 +110,7 @@ const BaseApp: React.FunctionComponent<{
 }> = ({staging, setStaging}) => {
   const [avalancheCenterId, setAvalancheCenterId] = React.useState(Constants.expoConfig.extra.avalanche_center as AvalancheCenterID);
   const [date] = React.useState<Date>(defaultDate);
-  const dateString = apiDateString(date);
+  const dateString = formatInTimeZone(date, 'UTC', 'yyyy-MM-dd HH:mm:ssXXX');
```

Comment:
why would you not extract this into a common helper? this seems like a maintenance headache for future you

Reply:
done :)